src/platforms/esp32/components/avm_builtins/idf_component.yml

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: AtomVM Contributors
+#
+## IDF Component Manager Manifest File
+dependencies:
+  # # Put list of dependencies here
+  # # For components maintained by Espressif:
+  # component: "~1.0.0"
+  # # For 3rd party components:
+  # username/component: ">=1.0.0,<2.0.0"
+  # username2/component2:
+  #   version: "~1.0.0"
+  #   # For transient dependencies `public` flag can be set.
+  #   # `public` flag doesn't have an effect dependencies of the `main` component.
+  #   # All dependencies of `main` are public by default.
+  #   public: true
+
+  espressif/esp_wifi_remote:
+    rules:
+     - if: "idf_version >= 5.4"
```

Comment:
My AI friend tells me we can do:
```
    rules:
      - if: "idf_version >= 5.4"
      - if: "target == 'esp32p4'"
```

not sure though..

Reply:
it does seem like we need to avoid loading esp_wifi_remote on other chips (for now/a start):
https://github.com/espressif/esp-wifi-remote/blob/main/components/esp_wifi_remote/idf_component.yml

it does also seem like we could add it to a special c6 build https://github.com/espressif/esp-wifi-remote/blob/main/components/esp_wifi_remote/README.md - and then have AtomVM on both chips. (and figure out disterl)

Reply:
`esp_wifi_remote` only supports esp43p4 target, and does this check internally, so this will not be included by other boards already.

> "it does also seem like we could add it to a special c6 build https://github.com/espressif/esp-wifi-remote/blob/main/components/esp_wifi_remote/README.md - and then have AtomVM on both chips. (and figure out disterl)"

That is a whole other can of worms and will require disterl over SDIO, and the P4 would not have native networking. This make WiFIi networking on the P4 identical to any other ESP32 with on board wifi. Also re-flashing the onboard C6 on all of P4/C6 boards requires an external uart-ttl-usb adapter, but they do come from the factory with esp_hosted firmware already.